packages/client/src/components/SafeSettings/components/Owners/AddSafeOwner.js

```diff
@@ -1,90 +1,111 @@
-        <p className="has-text-grey">
-          Heads up, they'll have full access to this safe
-        </p>
+        <h2 className="is-size-4 has-text-black">Add Owner</h2>
       </div>
-      <div className="border-light-top p-5 has-text-grey">
-        <div className="flex-1 is-flex is-flex-direction-column">
-          <label className="has-text-grey mb-2">Owner Name</label>
-          <input
-            className="p-4 rounded-sm border-light"
-            type="text"
-            placeholder="Enter local owner name"
-            value={name}
-            onChange={(e) => setName(e.target.value)}
-          />
-        </div>
-        <div className="flex-1 is-flex is-flex-direction-column mt-4">
-          <label className="has-text-grey mb-2">
-            Address<span className="has-text-red">*</span>
+      <div className="border-light-top border-light-bottom p-5 has-text-grey">
+        <div className="flex-1 is-flex is-flex-direction-column mb-4">
+          <label className="has-text-grey mb-2 with-icon is-flex is-align-items-center">
+            Address<span className="has-text-red">*</span> <Svg name="QuestionMark" className="ml-1" />
           </label>
           <div style={{ position: "relative" }}>
             <input
-              className="p-4 rounded-sm column is-full border-light"
+              className={addrInputClasses().join(" ")}
               type="text"
-              placeholder="Enter user's FLOW address"
+              placeholder=""
               value={address}
               onChange={(e) => onAddressChange(e.target.value)}
             />
             {addressValid && (
-              <div style={{ position: "absolute", right: 17, top: 14 }}>
-                <Svg name="Check" />
+              <div
+                style={{ position: "absolute", right: 17, top: 19 }}
+              >
+                <Svg name="Check" height="15" width="15" />
               </div>
             )}
           </div>
         </div>
-        <div className="is-flex is-align-items-center mt-6">
-          <button className="button flex-1 is-border mr-2" onClick={onCancel}>
+        <div className="flex-1 is-flex is-flex-direction-column">
+          <label className="has-text-grey mb-2 is-flex is-align-items-center">
+            Name <Svg name="QuestionMark"  className="ml-1" />
+          </label>
+          <input
+            className="input p-4 rounded-sm border-light is-size-6"
+            type="text"
+            placeholder=""
+            value={name}
+            onChange={(e) => setName(e.target.value)}
+          />
+        </div>
+      </div>
+      <div className="">
+        <div className="is-flex is-align-items-center p-5">
+          <button className="button flex-1 is-border mr-2 has-text-weight-bold" onClick={onCancelClick}>
             Cancel
           </button>
           <button
             disabled={!isAddressValid}
```

Comment:
Can you put here addressValid instead, now it's possible to click Next on the button